src/all/hentaifox/src/eu/kanade/tachiyomi/extension/all/hentaifox/HentaiFox.kt

```diff
@@ -143,6 +143,7 @@ class HentaiFox(
         }
         return xhrHeaders.newBuilder()
             .add("X-Csrf-Token", csrfToken)
+            .add("Referer", baseUrl)
```

Comment:
Minor thing, all browsers use a `/` suffix here.
```suggestion
            .add("Referer", "$baseUrl/")
```

Reply:
I checked from Postman and the correct url is the one without the trailing slash. I don't know if the slash is for some kind of manipulation/formal notation but as far as i can see both Postamn and Chrome DevTools agree that the slash shouldn't be there. If i am wrong lmk and i'll update it.

Reply:
Checking from Brave:
<img width="1310" height="700" alt="image" src="https://github.com/user-attachments/assets/40c87f7b-53e0-4d3a-81df-c5666efa9f9e" />

MDN example for the `Referer` also has a trailing slash for the root page: https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Referer#examples

Reply:
Ok, i was confidently incorrect then, my bad, i'll just merge your suggestion

Reply:
I even double checked, not sure what i saw, sorry once again